pkg/migration/controllers/migration.go

```diff
@@ -614,6 +705,20 @@ func (m *MigrationController) migrateResources(migration *stork_api.Migration) e
 	if err != nil {
 		return err
 	}
+	logrus.Infof("Cleaning up migrated resources")
```

Comment:
Don't think we need this. If you want to keep it use `log.MigrationLog(migration).Infof(...)`